.github/ISSUE_TEMPLATE/doc_improvement.md

```diff
@@ -0,0 +1,23 @@
+---

```

Comment:
This template can be removed as it's not used.

Reply:
Precisely, I was thinking it could be a good time to create a template for doc issues (and have the corresponding button), what do you think ?

Reply:
Sounds good to me.